internal/app/user/mongodb_repo.go

```diff
@@ -0,0 +1,120 @@
+package user
```

Comment:
Create a folder `repositories` in api folder, then move all the mongo_repo.go into it. Then rename this file to user_repo.go